govuk_admin_template.gemspec

```diff
@@ -23,8 +23,6 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "capybara", "~> 3"
   gem.add_development_dependency "govuk_test", "~> 2"
```

Comment:
I think govuk_test was only used for the selenium configuration for Jasmine, and so can be removed. My quick search didn't indicate that it was also used with capybara (which is presumably just through Rack) but I could be wrong.